src/redux/selectors/cellSets/getCellSets.js

```diff
@@ -2,6 +2,20 @@ import createMemoizedSelector from 'redux/selectors/createMemoizedSelector';
 
 import initialState from '../../reducers/cellSets/initialState';
 
-const getCellSets = () => (state) => (Object.keys(state).length ? state : initialState);
+const getCellSets = () => (state) => {
+  const stateToReturn = (Object.keys(state).length ? state : initialState);
+
+  const accessible = !(
+    stateToReturn.loading
+    || stateToReturn.initialLoadPending
+    || stateToReturn.updatingClustering
+    || stateToReturn.error
+  );
```

Comment:
I think the `!(...)` is very easy to miss. This is equal to `!stateToReturn.loading && ...` right? I think it's easier written that way.

Reply:
I agree, it's more clear also that none of the conditions should be true (using &&)
```
! stateToReturn.loading
&& !stateToReturn.initialLoadPending
&& !stateToReturn.updatingClustering
&& !stateToReturn.error
```